### DISCUSSION TITLE
Organize an Observability Community during Kubecon EU Paris

### DISCUSSION THREAD

**Discussion by henrikrexed** [cncf/tag-observability#155] [Ideas]
Since Kubecon Detroit, the observability community did not have the chance to meet in person.

We are enthusiastic about fostering collaboration and knowledge-sharing within the Observability space. To achieve this, we propose organizing a community event that brings together experts, enthusiasts, and practitioners to discuss the latest trends, best practices, and challenges in the Observability domain.

This discussion is aimed at agreeing on :
- format
- schedule
- and more

**Comment by henrikrexed** [comment]
I would suggest to organize a lunch or dinner during kubecon Paris. who is interested?

**Reply by alolita** [reply]
Good suggestion! Let's do it.

**Reply by mhausenblas** [reply]
Count me in!
